grants_tagger/models/mesh_cnn.py

```diff
@@ -20,6 +20,7 @@ def __init__(
         buffer_size=1000,
         data_cache=None,
         random_seed=42,
+        cutoff_prob=0.1,
```

Comment:
A philosophical question. Should you set defaults here at all?

Reply:
Given that otherwise Y_pred_proba does not fit to memory most of the time, this way it does and additionally most of low probs do not add any value.

Additionally sklearn has this philosophy of always having default if this is purely philosophical so you can initialize all classes by simply calling them e.g. SGDClassifier() or SVC()